x-pack/platform/plugins/shared/streams_app/public/components/stream_list_view/empty_prompt.tsx

```diff
@@ -26,6 +26,7 @@ export function StreamsEmptyPrompt() {
 
   return (
     <EuiEmptyPrompt
+      data-test-subj="streamsEmptyPrompt"
```

Comment:
Just a friendly heads up: we've defined this new test ID `streamsEmptyPrompt` but don't seem to be referencing it in the tests (yet).